internal/database/mysql.go

```diff
@@ -572,12 +604,17 @@ func (db *MySQLDatabase) GetAccounts(ctx context.Context, opts GetAccountsOption
 // Returns the account and boolean whether it's exist or not.
 func (db *MySQLDatabase) GetAccount(ctx context.Context, username string) (model.Account, bool, error) {
 	account := model.Account{}
-	if err := db.GetContext(ctx, &account, `SELECT
-		id, username, password, owner FROM account WHERE username = ?`,
-		username,
-	); err != nil {
-		return account, false, errors.WithStack(err)
-	}
+	row := db.QueryRowx(`SELECT
```

Comment:
You should be able to use the same `GetContext` here since `model.Account` implements [`driver.Scanner`](https://pkg.go.dev/database/sql#Scanner). Did you encounter any issues?

My guess is the same for all database engines.